Sources/Main/Services/FormPost.swift

```diff
@@ -16,47 +16,94 @@
 import Foundation
 import SwiftyJSON
 
-/// A struct representing a form POST request.
+/// An implementation of form POST ``Request``.
 public struct FormPost: Request {
- public typealias Response = AuthorizationRequest
-
- /// The HTTP method for the request.
- public var method: HTTPMethod { .POST }
-
- /// Additional headers to include in the request.
- public var additionalHeaders: [String: String] = [:]
-
- /// The URL for the request.
- public var url: URL
-
- /// The request body as data.
- public var body: Data? {
-   var formDataComponents = URLComponents()
-   formDataComponents.queryItems = formData.toQueryItems()
-   let formDataString = formDataComponents.query
-   
-   if let formDataString, formDataString.isEmpty {
-     return JSON(formData).rawString()?.data(using: .utf8)
```

Comment:
This functionality is not retained as I did not understand what was its purpose.